rules/cross-platform/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml

```diff
@@ -23,7 +23,7 @@ references = [
 risk_score = 73
 rule_id = "d22a85c6-d2ad-4cc4-bf7b-54787473669a"
 severity = "high"
-tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+tags = ["Elastic", "Host", "macOS", "Windows", "Threat Detection", "Defense Evasion"]
```

Comment:
this rule is only for macOS Operating syste, users can install MS Office for macOS to handle documents. don't think the `windows` tag fit well here.

Reply:
@Samirbous perhaps I should filter by host os in that case? The events that appeared were Windows events.